content/doc/book/system-administration/security/configuring-content-security-policy.adoc

```diff
@@ -0,0 +1,228 @@
+---
+title: Access Control for Builds
+layout: documentation
+---
+
+Jenkins 1.641 / Jenkins 1.625.3 introduce the
+`+Content-Security-Policy+` header to static files served by Jenkins
```

Comment:
No need for the backtick + plus combination for the vast majority of code formatted text in this document.